src/components/Highlights.vue

```diff
@@ -1,26 +1,181 @@
 <template>
   <div class="highlights-bar">
-    <div class="betas">JSE Index:{{ jseIndex }}</div>
-    <div class="betas">{{ industryIndices[1] }}: {{ industryBetas[1] }}</div>
-    <div class="betas">{{ industryIndices[2] }}: {{ industryBetas[2] }}</div>
+    <div class="betas">
+      All Share Index (30 Days): {{ index1 }}
+      <span
+        id="allShareNumber"
+        v-bind:class="{
+          positive: index1Change >= 0,
+          negative: !(index1Change >= 0),
+        }"
+        v-on:mount="index1Sign = index1Change <= 0"
+      >
+        ({{ index1Change }}%)
+      </span>
+    </div>
+
+    <div class="betas">
+      TOP 40 Index (30 Days): {{ index2 }}
+      <span
+        id="allShareNumber"
+        v-bind:class="{
+          positive: index2Change >= 0,
+          negative: !(index2Change >= 0),
+        }"
+        v-on:mount="index1Sign = index2Change <= 0"
+      >
+        ({{ index2Change }}%)
+      </span>
+    </div>
+
+    <div class="betas">
+      F&I Index (30 Days): {{ index3 }}
+      <span
+        id="allShareNumber"
+        v-bind:class="{
+          positive: index3Change >= 0,
+          negative: !(index3Change >= 0),
+        }"
+        v-on:mount="index3Sign = index3Change <= 0"
+      >
+        ({{ index3Change }}%)
+      </span>
+    </div>
+
+    <div class="betas">
+      Industrials Index (30 Days): {{ index4 }}
+      <span
+        id="allShareNumber"
+        v-bind:class="{
+          positive: index4Change >= 0,
+          negative: !(index4Change >= 0),
+        }"
+        v-on:mount="index4Sign = index4Change <= 0"
+      >
+        ({{ index4Change }}%)
+      </span>
+    </div>
+
+    <div class="betas">
+      Resources Index (30 Days): {{ index5 }}
+      <span
+        id="allShareNumber"
+        v-bind:class="{
+          positive: index5Change >= 0,
+          negative: !(index5Change >= 0),
+        }"
+        v-on:mount="index5Sign = index5Change <= 0"
+      >
+        ({{ index5Change }}%)
+      </span>
+    </div>
   </div>
 </template>
 
 <script>
+//Axios stuff
+import axios from 'axios'
+
+var period = 22
```

Comment:
Don't use var. Use `const` or a `let` (in this case `const`) https://www.freecodecamp.org/news/var-let-and-const-whats-the-difference/

Also this isn't necessary since you're defining period in data if I'm not mistaken

Reply:
I thought so too, but whenever I remove it I get this:
![image](https://user-images.githubusercontent.com/80746660/121642584-02254e00-ca91-11eb-9a74-93f581c0cf14.png)